### PR TITLE
feat: modifies API to add more features to superbowl

### PR DESCRIPTION
**Pull request by JonathanGriffe**

### CODE REVIEW

**Review by antoinerabany — CHANGES_REQUESTED** [review]

File: remoulade/helpers/actor_arguments.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        try:
+            return str(rawtype.__name__)
+        except AttributeError:
+            return str(rawtype)
+
+    args = []
+    for param in params.values():
+        arg = {"name": param.name}
+        if param.annotation != Parameter.empty:
+            arg["type"] = parsetype(param.annotation)
```
Comment (SimonDosda) [inline remoulade/helpers/actor_arguments.py:19]: `parse_type`

File: remoulade/helpers/actor_arguments.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        if param.default != Parameter.empty:
+            arg["default"] = str(param.default)
```
Comment (SimonDosda) [inline remoulade/helpers/actor_arguments.py:21]: does it work for `None` ?

Comment (JonathanGriffe) [reply remoulade/helpers/actor_arguments.py:21]: I added a test to make sure it does. The only problem is that there is no way to differentiate between None and "None" as it is parsed as JSON.

Comment (SimonDosda) [reply remoulade/helpers/actor_arguments.py:21]: I was thinking about replacing it by an empty string, as this is what you usually want (but we will have to see how to manage None values anyway)

**Review by SimonDosda — APPROVED** [review]
ok for me, can you just add the changelog considering these changes as a feature

**Pull request merged**